src/librustc_cratesio_shim/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+# This crate exists to allow rustc to link certain crates from crates.io into
+# the distribution. This doesn't work normally because:
+#
+# - Cargo always builds dependencies as rlibs:
+#   https://github.com/rust-lang/cargo/issues/629
```

Comment:
There's not really a bug here, the manifests say they want to get built as rlibs.

Reply:
I didn't say it was a bug, but also I'm pretty sure bitflags' manifest doesn't specify one way or the other. https://github.com/rust-lang-nursery/bitflags/blob/master/Cargo.toml